src/store/modules/target.js

```diff
@@ -994,6 +1046,50 @@ export default ({
         sender: serverNote.createdBy
       }
       return localNote
+    },
+    mapSubdomainsFromServerToLocal: (state, getters) => (subdomains) => {
+      const mappedSubdomains = []
+      subdomains.forEach(subdomain => {
+        const mappedSubdomain = getters.mapSingleSubdomainFromServerToLocal(subdomain)
+        mappedSubdomains.push(mappedSubdomain)
+      })
+      return mappedSubdomains
+    },
+    mapSingleSubdomainFromServerToLocal: (state) => (subdomain) => {
+      const createdDate = new Date(subdomain.createdAt)
+      const newSubDomain = {
+        id: subdomain.id,
+        name: subdomain.name,
+        added: createdDate.getFullYear() + '-' + (createdDate.getMonth() + 1) + '-' + createdDate.getDate(),
+        checking: false,
+        interesting: false,
+        vulnerable: false,
+        boubty: false,
```

Comment:
```
        checking: false,
        interesting: false,
        vulnerable: false,
        bounty: false,
        ignore: false,
```
is part of an array of labels, not separate fields
https://github.com/reconness/reconness/blob/7fd8fa03ef76ae3c2e01323d843ffb96ddd78374/src/ReconNess.Entities/Subdomain.cs#L26